swift_scality_backend/server.py

```diff
@@ -51,7 +51,7 @@ def get_diskfile(self, device, partition, account, container, obj,
 
     def async_update(self, op, account, container, obj, host, partition,
                      contdevice, headers_out, objdevice,
-                     policy_index=POLICY_IDX_STUB):
+                     policy=POLICY_STUB):
```

Comment:
Not a big fan of this name change, but not entirely sure how to work around it either...